### ISSUE TITLE
Dropdown child role of textbox not present and causes a11y health check tests to fail

### ISSUE THREAD

**Issue by Alex1100** [microsoft/fluentui#18107]
- **Package version(s)**: (fill this out)
"@fluentui/react@^7.153.4":
  version "7.170.0"
  resolved "https://msasg.pkgs.visualstudio.com/_packaging/ProductInsights/npm/registry/@fluentui/react/-/react-7.170.0.tgz#9dc78011050d74b549b74b630f60e2fde41918c9"
  integrity sha1-nceAEQUNdLVJt0tjD2Di/eQZGMk=
  dependencies:
    "@uifabric/set-version" "^7.0.24"
    office-ui-fabric-react "^7.170.0"
    tslib "^1.10.0"

 Validation result:
    [
      {
        "id": "aria-required-children",
        "impact": "critical",
        "tags": [
          "cat.aria",
          "wcag2a",
          "wcag131"
        ],
        "description": "Ensures elements with an ARIA role that require child roles contain them",
        "help": "Certain ARIA roles must contain particular children",
        "helpUrl": "https://dequeuniversity.com/rules/axe/3.5/aria-required-children?application=axeAPI",
        "nodes": [
          {
            "any": [
              {
                "id": "aria-required-children",
                "data": [
                  "textbox"
                ],
                "relatedNodes": [],
                "impact": "critical",
                "message": "Required ARIA child role not present: textbox"
              }
            ],
            "all": [],
            "none": [],
            "impact": "critical",
            "html": "<div data-is-focusable=\"true\" id=\"Dropdown1\" tabindex=\"0\" role=\"combobox\" aria-haspopup=\"listbox\" aria-expanded=\"false\" aria-labelledby=\"Dropdown1-label Dropdown1-option\" class=\"ms-Dropdown dropdown-40\">",
            "target": [
              "#Dropdown1"
            ],
            "failureSummary": "Fix any of the following:\n  Required ARIA child role not present: textbox"
          }
        ]
      }
    ]

**Comment by TristanWatanabe** [comment]
Hey @Alex1100, can you provide repro steps in the form of a codepen or codesandbox for the behavior above?
